moped-editor/src/layouts/DashboardLayout/Footer.js

```diff
@@ -27,8 +27,10 @@ const Footer = () => {
       "https://api.github.com/repos/cityofaustin/atd-moped/tags";
     const response = await fetch(githubTagsApiEndpoint);
     const data = await response.json();
-    if (data) {
+    if (data[0]) {
       setMostRecentTag(data[0].name);
```

Comment:
this line was erroring when the response from the api endpoint included an error message. There was data being returned, but not an array of tags.